etc/profile-m-z/mattermost-desktop.profile

```diff
@@ -0,0 +1,37 @@
+# Firejail profile for mattermost-desktop
+# This file is overwritten after every install/update
+# Persistent local customizations
+include mattermost-desktop.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/Mattermost
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-programs.inc
+include disable-passwdmgr.inc
+
+mkdir ${HOME}/.config/Mattermost
+whitelist ${DOWNLOADS}
+whitelist ${HOME}/.config/Mattermost
+include whitelist-common.inc
+include whitelist-var-common.inc
+
+caps.drop all
+netfilter
+nodvd
+nogroups
+notv
+nou2f
+novideo
+noroot
+shell none
+protocol unix,inet,inet6
+
+disable-mnt
+private-dev
+private-tmp
+private-etc machine-id,resolv.conf,fonts
```

Comment:
- `ca-certificates,ssl,pki,crypto-policies,nsswitch.conf`??
- sort alphabetical.
- Move above `private-tmp`

-----

`dbus-user none`?
`dbus-system none`

Reply:
too old firejail :(

Reply:
You can test with `blacklist /run/dbus` (dbus-system none) and `blacklist ${RUNUSER}/bus`.

The flatpak has the following policy, so (native) notifications will be broken.
```
dbus-user filter
dbus-user.own com.mattermost.Desktop
dbus-user.talk org.freedesktop.Notifications
dbus-system none
```

I would say add the dbus-user rules commented.